server/channels/app/export.go

```diff
@@ -125,6 +125,12 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, job
 		job.Data = make(model.StringMap)
 	}
 
+	if job == nil {
+		job = &model.Job{
+			Data: make(model.StringMap),
+		}
+	}
```

Comment:
A couple of questions:

1. When would the job be `nil`?
2. Should we perhaps consolidate to one call only by reworking the above check? (lines 124-125).